crates/rover-client/src/shared/check_response.rs

```diff
@@ -10,42 +10,80 @@ use serde::{Deserialize, Serialize};
 use prettytable::{row, Table};
 use serde_json::{json, Value};
 
+#[derive(Debug, Serialize, Clone, Eq, PartialEq)]
+pub enum CheckResponse {
+    OpeartionCheckResponse(OperationCheckResponse),
+    OpeartionLessCheckResponse(OpeartionLessCheckResponse),
+}
+
+impl CheckResponse {
+    pub fn get_json(&self) -> Value {
+        match self {
+            CheckResponse::OpeartionCheckResponse(operation_check_response) => {
+                operation_check_response.get_json()
+            }
+            CheckResponse::OpeartionLessCheckResponse(operation_less_check_response) => {
+                operation_less_check_response.get_json()
+            }
+        }
+    }
+}
+
+#[derive(Debug, Serialize, Clone, Eq, PartialEq)]
+pub struct OpeartionLessCheckResponse {
+    pub target_url: Option<String>,
+    pub core_schema_modified: bool,
+    pub contains_operation_check: bool,
+}
+
+impl OpeartionLessCheckResponse {
+    pub fn to_output(&self) -> String {
+        "".to_string()
```

Comment:
is this correct?

Reply:
Yep. That is incorrect thanks for catching.